CryptoTechProject.Tests/AcceptanceTest.cs

```diff
@@ -67,10 +67,13 @@ public void CanGetTwoAirtableWorkshops()
             GetWorkshops getWorkshops = new GetWorkshops(airtableGateway);
             GetWorkshopsResponse response = getWorkshops.Execute();
 
-            DateTime time = new DateTime(2019, 10, 18, 14, 00, 0);
+            DateTime sourceDate = new DateTime(2019, 10, 18, 14, 00, 0);
+            DateTimeOffset time = new DateTimeOffset(sourceDate, TimeZoneInfo.FindSystemTimeZoneById("Europe/London").GetUtcOffset(sourceDate));
             
-            DateTime time2 = new DateTime(2019, 10, 18, 15, 30, 0);
             
+            DateTime sourceDate2 = new DateTime(2019, 10, 18, 15, 30, 0);
```

Comment:
Is there a specific reason why you chose the datetimes given here?

Reply:
We were just looking for a value to satisfy the test and make it pass. It was a value taken from other tests we have in Acceptance tests. 

We know we must correct it to a more general solution that handles the error correctly. 

We don't want a user to not enter their own Time requirements and have this default value inserted instead.